ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -996,6 +996,13 @@ keys:
         transceiver:
           type: dict
           keys:
+            frequency:
+              type: int
+              convert_types:
+                - str
+              min: 190000
+              max: 200000
+              description: Transceiver Laser Frequency for 400GBASE-ZR in GHz.
```

Comment:
```suggestion
              description: Transceiver Laser Frequency in GHz.
```

Reply:
Since changing to a string will not support min/max, you can add the min-max in the description.